### PR TITLE
DOCS-9458-email-send-editorial-kt'

### PR DESCRIPTION
**Pull request by kevintroller**
Email 1.3 email-send.adoc editorial

### CODE REVIEW

File: email/1.3/modules/ROOT/edits/email-send.adoc [hunk 1]
```diff
@@ -0,0 +1,363 @@
+= Send Emails with Email Connector Examples - Mule 4
+:keywords: email, connector, configuration, smtp, send, smtps
+:page-aliases: connectors::email/email-send.adoc
+
+Anypoint Connector for Email (Email Connector) can send messages over SMTP and SMTPS servers by using the *Send* operation. The following examples show how to configure Email Connector to send emails and attachments, in both Anypoint Studio and the XML editor:
```
Comment (kbostrom) [inline email/1.3/modules/ROOT/edits/email-send.adoc]: Minor point, but I wonder if we should reword this to put the audience into the action:
```suggestion
Use Anypoint Connector for Email (Email Connector) to send messages over SMTP and SMTPS servers by using the *Send* operation. The following examples show how to configure Email Connector to send emails and attachments, in both Anypoint Studio and the XML editor:
```

File: email/1.3/modules/ROOT/edits/email-send.adoc [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+.Email Connector Send messages over SMTPS
+image::email-send-flow.png[Email Connector Send messages over SMTPS flow]
+
+To create the flow: +
+
+. Create a new Mule project in Studio.
+. In the *Mule Palette* view, select the *Scheduler* component and drag it onto the canvas. +
+This source initiates a flow when a time-based condition is met.
+. Drag the Email *Send* operation to the right of the *Scheduler* source. +
+. On the *Send* configuration screen, click the plus sign (*+*) next to the *Connector configuration* field to configure a global element for the operation.
```
Comment (kbostrom) [inline email/1.3/modules/ROOT/edits/email-send.adoc]: "on the screen"?
GLOBAL edit in other files in this set, re. "on" vs. "in"

Comment (kbostrom) [reply email/1.3/modules/ROOT/edits/email-send.adoc]: Actually, I am going to review this document and add what I think is relevant to our style ref, so stand by....:
https://developers.google.com/style/ui-elements

File: email/1.3/modules/ROOT/edits/email-send.adoc [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+.Email SMTPS configuration
+image::email-smtps-configuration.png[Email SMTPS configuration]
+
+[start=12]
+. In the *Send* operation configuration screen, set the following values:
++
+* *From address* +
+`user1@domain.com`
+* *To addresses* +
+`Edit inline`
```
Comment (kbostrom) [inline email/1.3/modules/ROOT/edits/email-send.adoc]: I'm still thinking about this formatting, and I am asking for input....

The style ref. recommends inserting a blank line space between a list item and its description, which is perfectly fine in most cases and also helps with scanning and thinking about the elements as list item + separate paragraph.
But in lists like this one, that recommendation results in a lot of space:

```suggestion
. In the *Send* operation configuration screen, set the following values:
+
* *From address* +
+
`user1@domain.com`
* *To addresses* +
+
`Edit inline`
```
I hate to break my own rules, but when the description IS THIS SHORT, I think we might actually want to use this format (which I would never recommend in any other situation):

```suggestion
. In the *Send* operation configuration screen, set the following values:
+
* *From address*: `user1@domain.com`
* *To addresses*: `Edit inline`
```
What do YOU think?

Comment (kbostrom) [reply email/1.3/modules/ROOT/edits/email-send.adoc]: Sorry! Back to COPYEDITING ONLY....

File: email/1.3/modules/ROOT/edits/email-send.adoc [hunk 4]
```diff
@@ -0,0 +1,363 @@
...
++
+[start=13]
+. Click the plus sign (*+*) to add new emails.
+. Set the *Value* field to `user2@domain.com`.
+. Repeat the previous two actions and add another email as `user3@domain.com`.
```
Comment (kbostrom) [inline email/1.3/modules/ROOT/edits/email-send.adoc]:
```suggestion
. Repeat the previous two actions but set the value to `user3@domain.com`.
```

File: email/1.3/modules/ROOT/edits/email-send.adoc [hunk 5]
```diff
@@ -0,0 +1,363 @@
...
+. Set the *Subject* field to `IMPORTANT!`.
+. Set the *Content* field to `"<h1>Hello this is an important message</h1>"` and the *ContentType* field to `text/html`. +
+If no body *Content* is specified, the content of the message payload is used by default. If that payload is not `text`, the operation fails with an `EMAIL:SEND` error.
```
Comment (kbostrom) [inline email/1.3/modules/ROOT/edits/email-send.adoc]:
```suggestion
If *Content* is not empty, the content of the message payload is used by default. If that payload is not `text`, the operation fails with an `EMAIL:SEND` error.
```
OR
```suggestion
If no body content is specified, the content of the message payload is used by default. If that payload is not `text`, the operation fails with an `EMAIL:SEND` error.
```

File: email/1.3/modules/ROOT/edits/email-send.adoc [hunk 6]
```diff
@@ -0,0 +1,363 @@
...
+		</email:send>
+	</flow>
+</mule>
+----
+
+
+[[send-attachments]]
+== Send an Attachment
+
+The following example illustrates how to send emails and attachments over the SMTP server. Use DataWeave to handle the attachments. The flow reads a JSON file using the File Connector *Read* operation and then uses the Email Connector *Send* operation to send the content of the file as an attachment:
```
Comment (kbostrom) [inline email/1.3/modules/ROOT/edits/email-send.adoc]: "handle" > "process"?
I typically recommend not using "handle" unless within a context that mentions "handlers." Otherwise, the term can be temporarily misleading to nonnative English speakers.

File: email/1.3/modules/ROOT/edits/email-send.adoc [hunk 7]
```diff
@@ -0,0 +1,363 @@
...
+
+The following screenshot shows the Anypoint Studio app flow for this example:
+
+.Email Connector Send attachments over SMTP
+image::email-attachment-flow.png[Email Connector Send attachments over SMTP]
+
+To create the flow: +
+
+. Create a new Mule project in Studio.
+. In the *Mule Palette* view, select the *Scheduler* component and drag it on to the canvas. +
```
Comment (kbostrom) [inline email/1.3/modules/ROOT/edits/email-send.adoc]:
```suggestion
. In the *Mule Palette* view, select the *Scheduler* component and drag it onto the canvas. +
```

File: email/1.3/modules/ROOT/edits/email-send.adoc [hunk 8]
```diff
@@ -0,0 +1,363 @@
...
+`995`
+* *User* +
+`user1@domain.com`
+* *Password* +
+`password`
++
+[start=9]
+. Click *OK*.
+. In the *Send* operation configuration screen, set the *To addresses* field to `Edit inline`.
+. Click the plus sign (*+*) to add a new email *Value* as `example@domain.com`.
```
Comment (kbostrom) [inline email/1.3/modules/ROOT/edits/email-send.adoc]:
```suggestion
. Click the plus sign (*+*) to add a new email value, `example@domain.com`.
```

File: email/1.3/modules/ROOT/edits/email-send.adoc [hunk 9]
```diff
@@ -0,0 +1,363 @@
...
+        }]</email:attachments>
+        </email:send>
+	</flow>
+</mule>
+----
+
+
+== Send Multiple Attachments
+
+The following example illustrates how to send emails and multiple attachments over the SMTP server. The flow initiates with a *Scheduler* component. Then, *Set Variable* components define each attachment media type JSON, text and file. Subsequently, the Email Connector *Send* operation sends the attachments in the email. Then log the result message using a *Logger* component.
```
Comment (kbostrom) [inline email/1.3/modules/ROOT/edits/email-send.adoc]:
```suggestion
The following example illustrates how to send emails and multiple attachments over the SMTP server. The flow initiates with a *Scheduler* component. Then, *Set Variable* components define each attachment media type JSON, text and file. Subsequently, the Email Connector *Send* operation sends the attachments in the email, after which you log the result message using a *Logger* component.
```

File: email/1.3/modules/ROOT/edits/email-send.adoc [hunk 10]
```diff
@@ -0,0 +1,363 @@
...
+
+The following screenshot shows the Anypoint Studio app flow for this example:
+
+.Email Connector Multiple attachments
+image::email-multipleattachment-flow.png[Email Connector Multiple attachments]
+
+To create the flow: +
+
+. Create a new Mule project in Studio.
+. In the *Mule Palette* view, select the *Scheduler* component and drag it on to the canvas. +
```
Comment (kbostrom) [inline email/1.3/modules/ROOT/edits/email-send.adoc]:
```suggestion
. In the *Mule Palette* view, select the *Scheduler* component and drag it onto the canvas. +
```

**Review by kbostrom — DISMISSED** [review]
CE per request

**Review by dhirsch1 — APPROVED** [review]
